README.md

```diff
@@ -22,12 +22,46 @@ The script serves as installer by default, but can also be used with flags `--up
 
 The agent can run as a standalone binary, as long as configuration files are set up correctly.
 
+### Using a Docker reverse proxy
+If you'd like to avoid adding `metrikad` user to the `docker` group you can alternatively use a reverse proxy to filter traffic before it reaches the non-networked Docker socket and avoid having the agent process speaking directly to it. With this setup, the reverse proxy must be still run by a user that belongs to the `docker` group, so that it can proxy requests to the UNIX socket.
```

Comment:
@iogakos minor suggestion on some of the copy here: 

To avoid adding the `metrikad` user to the `docker` group, you can use a reverse proxy. The reverse proxy filters traffic before it reaches the non-networked Docker socket, preventing the agent process from speaking directly to it. The reverse proxy still needs a user that belongs to the `docker` group to run it so it can proxy requests to the UNIX socket.

Reply:
Addressed in 77efbb6.